change/@fluentui-react-popover-8d4e3d2c-363f-4aa5-bfde-adceecb01f68.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29894&commitId=6357182f30e6127ba8ec1fcba41caaa4862c404c&env=prod&clientType=FLUENTUI)


  <details><summary><b>AvatarGroup Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|1598001| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|1553663| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|1552205| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|310371| Changed|

</details>


  <details><summary><b>Popover Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.when rendering inline, it should not render behind relatively positioned elements.PopoverSurface focused.chromium.png|1599588| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|510843| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|261018| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|1590190| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|510861| Changed|

</details>


  <details><summary><b>Portal</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Portal.Apply Class Names.should have green border.chromium.png|1600220| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>